protocol-models/src/main/resources/airbyte_protocol/v0/airbyte_protocol.yaml

```diff
@@ -429,7 +429,10 @@ definitions:
         items:
           "$ref": "#/definitions/SyncMode"
       source_defined_cursor:
-        description: If the source defines the cursor field, then any other cursor field inputs will be ignored. If it does not, either the user_provided one is used, or the default one is used as a backup.
+        description: |-
```

Comment:
I thought the yaml line break was just a `|` not a `|-`?

Reply:
`-` strips newlines from the end.

I started by pattern matching existing descriptions. I do agree stripping trailing newlines is good.

Reply:
Lmk if you disagree!